src/js/helpers.js

```diff
@@ -255,11 +250,6 @@ export function stringifyFormReplacer(key, value) {
       return undefined;
     }
 
-    // autosuggest widgets save value and label info, but we should just return the value
```

Comment:
I don't know if this is a special case as much as it is the way our widget is written; we may not be able to remove this, because otherwise the wrong data will be submitted for every instance of the autosuggest field. A more permanent fix might be changing what the widget returns, but for now I think this needs to stay?

Reply:
Does the "we should" in the comment represent what every library user would want from the widget? (In other words, nobody wants the labels.) If so we should change the widget so this transformation isn't necessary. If this is more about how vets.gov would like to see the info at their back-end, they can add their own custom transformation. Either way it's a breaking change.

Reply:
Good question; I'm not sure! I guess we don't know until we get other users. Given our only current user wants only the values, I would advocate for keeping this transformation in here, or at least changing the widget so this transformation isn't necessary. But curious to hear what you think!

Reply:
They can still have the values the way they want them, by doing their own modification of the data before submit. It sounds like vets.gov will need that function already if they still need that special case for addresses. 

Think about it this way, if we add more widgets would we add special case code for those widgets here as well? It doesn't seem so. Either the widget would produce the "right" format in the form or we'd ask users to transform it themselves before submitting if they prefer a different format.